content/pages/pension/apply.md

```diff
@@ -5,6 +5,12 @@ title: Pension Benefits Application Process
 concurrence: ""
 order: 2
 children: pensionApplication
+widgets:
+  - root: react-pensionApplicationStatus
+    timeout: 20
+    loadingMessage: Checking your application status.
+    slowMessage: Sorry, this is taking longer than expected.
```

Comment:
This is probably going to be used a lot; can it just be a default message that can be overridden by supplying this parameter?